packages/react-query/src/internals/context.tsx

```diff
@@ -98,45 +98,37 @@ export interface TRPCContextState<
    */
   fetchQuery: (
     queryKey: TRPCQueryKey,
-    opts?: TRPCFetchQueryOptions<unknown, TRPCClientError<TRouter>, unknown>,
+    opts?: TRPCFetchQueryOptions<unknown, TRPCClientError<TRouter>>,
   ) => Promise<unknown>;
   /**
    * @link https://tanstack.com/query/v4/docs/reference/QueryClient#queryclientfetchinfinitequery
    */
   fetchInfiniteQuery: (
     queryKey: TRPCQueryKey,
-    opts?: TRPCFetchInfiniteQueryOptions<
-      unknown,
-      TRPCClientError<TRouter>,
-      unknown
-    >,
```

Comment:
these seem to have been wrong all along but i guess the bug never surfaced since this isn't very used and `getNextPageParam` wasn't required before?